src/Microsoft.IdentityModel.Tokens/Validation/Validators.Algorithm.cs

```diff
@@ -37,7 +37,7 @@ public static partial class Validators
         /// <param name="validationParameters"><see cref="ValidationParameters"/> required for validation.</param>
         /// <param name="callContext"></param>
 #pragma warning disable CA1801 // TODO: remove pragma disable once callContext is used for logging
-        internal static AlgorithmValidationResult ValidateAlgorithm(
+        internal static Result<string, ExceptionDetail> ValidateAlgorithm(
             string algorithm,
             SecurityKey securityKey,
```

Comment:
Do we still need SecurityKey and SecurityToken parameters?

Reply:
Addressed in another related comment.